cli/util/convert.go

```diff
@@ -70,16 +70,10 @@ func NewCommands() []*cli.Command {
 			Value:  neofs.DefaultBlockAttribute,
 			Action: cmdargs.EnsureNotEmpty("block-attribute"),
 		},
-		&cli.StringFlag{
-			Name:   "index-attribute",
-			Usage:  "Attribute key of the index file object",
-			Value:  neofs.DefaultIndexFileAttribute,
-			Action: cmdargs.EnsureNotEmpty("index-attribute"),
-		},
 		&cli.UintFlag{
-			Name:  "index-file-size",
-			Usage: "Size of index file",
-			Value: neofs.DefaultIndexFileSize,
+			Name:  "batch-size",
+			Usage: "Size of the batch of blocks to upload in parallel",
+			Value: neofs.DefaultBatchSize,
```

Comment:
Some note should be added that `batch-size` must be consistent from run to run, otherwise this logic won't work correctly.